tests/integration/docusaurus/expectations/creating_custom_expectations/expect_table_columns_to_be_unique.py

```diff
@@ -0,0 +1,226 @@
+                    "include_in_gallery": True,
+                    "in": {"strict": True},
+                    "out": {"success": False},
+                },
+            ],
+        },
+    ]
+
+    # This is a tuple consisting of all Metrics necessary to evaluate the Expectation.
+    metric_dependencies = ("table.columns.unique", "table.columns")
+
+    # This a tuple of parameter names that can affect whether the Expectation evaluates to True or False.
+    success_keys = ("strict",)
+
+    # This dictionary contains default values for any parameters that should have default values.
+    default_kwarg_values = {"strict": True}
+
+    def validate_configuration(self, configuration: Optional[ExpectationConfiguration]):
+        """
+        Validates that a configuration has been set, and sets a configuration if it has yet to be set. Ensures that
+        necessary configuration arguments have been provided for the validation of the expectation.
+
+        Args:
+            configuration (OPTIONAL[ExpectationConfiguration]): \
+                An optional Expectation Configuration entry that will be used to configure the expectation
+        Returns:
+            True if the configuration has been validated successfully. Otherwise, raises an exception
+        """
+
+        super().validate_configuration(configuration)
+        if configuration is None:
+            configuration = self.configuration
+
+        strict = configuration.kwargs.get("strict")
+
+        # Check other things in configuration.kwargs and raise Exceptions if needed
+        try:
+            assert (
+                isinstance(strict, bool) or strict is None
+            ), "strict must be a boolean value"
+        except AssertionError as e:
+            raise InvalidExpectationConfigurationError(str(e))
+
+        return True
+
+    # This method performs a validation of your metrics against your success keys, returning a dict indicating the success or failure of the Expectation.
+    def _validate(
+        self,
+        configuration: ExpectationConfiguration,
+        metrics: Dict,
+        runtime_configuration: dict = None,
+        execution_engine: ExecutionEngine = None,
+    ):
+        unique_columns = metrics.get("table.columns.unique")
+        table_columns = metrics.get("table.columns")
+        strict = configuration.kwargs.get("strict")
+
+        duplicate_columns = unique_columns.symmetric_difference(table_columns)
+
+        if strict is True:
+            success = len(duplicate_columns) == 0
+        else:
+            success = len(duplicate_columns) < len(table_columns)
+
+        return {
+            "success": success,
+            "result": {"observed_value": {"duplicate_columns": duplicate_columns}},
+        }
+
+    # This dictionary contains metadata for display in the public gallery
+    library_metadata = {
+        "tags": ["uniqueness"],
+        "contributors": ["@joegargery"],
+    }
+
+
+# </snippet>
```

Comment:
why are there blank lines above this snippet tag and the others?

Reply:
`black` does this. I hate it, but I hate failing linting tests more.